templates/style.css

```diff
@@ -525,3 +525,15 @@ body.darkmode div pre {
 body.darkmode span.tag {
     color: #FFD119;
 }
+
+body.darkmode .keyword {
+    color: chocolate;
```

Comment:
Thanks, but can we provide hex-color codes here instead of strings?

Reply:
Have updated with color code and also replace toggle buttons with moon/sun icon.

Reply:
Light mode
![image](https://user-images.githubusercontent.com/6072775/170945771-e9fc5240-f7dc-4a2c-8f79-8e65adef7015.png)

Darkmode
![image](https://user-images.githubusercontent.com/6072775/170945823-dce96594-3e2a-4fd8-9ff7-a847c6c4af1e.png)